CHANGELOG

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+## [v1.38.4] - 2019-04-07
+### Added
+- Elasticsearch Client abstraction API to mozdef-util for opening or closing indices.
```

Comment:
We do not need to modify this file right now, this is only when we do a mozdef release.

We only need to modify https://github.com/mozilla/MozDef/blob/master/mozdef_util/HISTORY.rst when we're doing a mozdef-util release, which you have done correctly.